Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -193,6 +188,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Issue9355.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Issue8784.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Issue9360.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Issue9440.cs" />
```

Comment:
This file still doesn't quite seem correct. This should be the only modified liine

Reply:
@PureWeen I added this line because of a merge conflict. So I assume that I need to accept changes from my branch and the target one. Am I correct thinking that way?